docs/GenericsManifesto.md

```diff
@@ -272,6 +283,8 @@ Unlike the minor extensions, major extensions to the generics model provide more
 
 ### Conditional conformances (*)
 
+*This feature has been accepted in [SE-0143](https://github.com/apple/swift-evolution/blob/master/proposals/0143-conditional-conformances.md) and is planned for release in SWIFT_VERSION_HERE.*
```

Comment:
We haven't started implementing conditional conformances yet so definitely not 3.1; again, we can't commit to anything showing up in any future release.